src/fswAlgorithms/orbitControl/lambertSolver/lambertSolver.cpp

```diff
@@ -80,6 +80,58 @@ void LambertSolver::readMessages(){
     }
 }
 
+/*! This method writes the output messages each call of updateState
+    @param CurrentSimNanos current simulation time in nano-seconds
+    @return void
+*/
+void LambertSolver::writeMessages(uint64_t CurrentSimNanos){
+    // Make local copies of messages
+    LambertSolutionMsgPayload lambertSolutionOutMsgBuffer;
+    LambertPerformanceMsgPayload lambertPerformanceOutMsgBuffer;
+    // Always zero the output message buffers before assigning values
+    lambertSolutionOutMsgBuffer = this->lambertSolutionOutMsg.zeroMsgPayload;
+    lambertPerformanceOutMsgBuffer = this->lambertPerformanceOutMsg.zeroMsgPayload;
+
+    if (this->noSolution || (this->M > 0 && !this->multiRevSolution)){
+        // 1. if the transfer angle is 180 degrees, the two position vectors do not define a plane, so an infinite number of solutions exist. In this case, the module should not return any solutions.
+        // 2. transfer time is too short for multi-revolution solution, zero solutions exist. Neither solution is valid
+        lambertSolutionOutMsgBuffer.valid = 0;
+        lambertSolutionOutMsgBuffer.valid_sol2 = 0;
+    }
+    else if (this->M == 0){
+        // if zero orbits are completed, only one solution exists. Only populate information for 1st solution, 2nd solution remains zero message payload
+
```

Comment:
:large_blue_circle: Extra new line can be deleted.

Reply:
Done.